build.rs

```diff
@@ -12,30 +12,9 @@ fn main() {
     if cfg!(feature = "llvm-11") {
         versions.push(11);
     }
-    let selected_version = match versions.len() {
+    match versions.len() {
         0 => panic!("llvm-ir: Please select an LLVM version using a Cargo feature."),
         1 => versions[0],
```

Comment:
tiny note, but since the `let selected_version =` was removed, we can also make this line be `1 => (),` or similar.